Utilities/Xerces/interface/XercesStrUtils.h

```diff
@@ -0,0 +1,65 @@
+#ifndef UTILITIES_XERCES_STRING_UTILS_H
+#define UTILITIES_XERCES_STRING_UTILS_H
+
+#include <xercesc/util/XercesDefs.hpp>
+#include <xercesc/util/XMLString.hpp>
+
+#ifdef XERCES_CPP_NAMESPACE_USE
+XERCES_CPP_NAMESPACE_USE
+#endif
+
+namespace cms {
+  namespace utils {
```

Comment:
Good use of namespaces. My one suggestion is to use `cms::xerces` as the namespace since this is only useful for Xerces.